eth_tester/utils/backend_testing.py

```diff
@@ -264,8 +264,10 @@ def test_mine_multiple_blocks(self, eth_tester):
     #
     @pytest.mark.parametrize('is_pending', [True, False])
     def test_send_raw_transaction_valid_raw_transaction(self, eth_tester, is_pending):
-        if is_pending and not hasattr(eth_tester.backend, 'send_signed_transaction'):
-            pytest.xfail("backend does not support presigned pending transactions")
+        if is_pending:
+            from eth_tester.backends import PyEthereum16Backend, PyEthereum21Backend
+            if isinstance(eth_tester.backend, (PyEthereum16Backend, PyEthereum21Backend)):
+                pytest.xfail("backend does not support presigned pending transactions")
```

Comment:
Can you add `xfail=strict` to the `pytest.ini` for this project so that these and any other `xfail` tests don't accidentally pass.

Reply:
Blarg, no big deal.  My internet connection must have dropped when I initially submitted this comment.